app/dashboard/templates/yge/base.html

```diff
@@ -45,7 +45,7 @@
   </head>
   <body id="yge" class="{{ class }}">
     {% include 'shared/tag_manager_2.html' %}
-    {% include 'shared/analytics.html' with suppress_sumo=1 %}
+    {% include 'shared/analytics.html' with suppress_sumo=0 %}
```

Comment:
You can remove the `supress_sumo` just to display it !

Reply:
this is suppressing sumo on the youvegoteth (tips flow).. it doesnt have anything to do with what i perceive to be the issue (sumo not showing on the landing page)

Reply:
The sumo does appear on the landing page currently though!